universal-application-tool-0.0.1/conf/application.test.conf

```diff
@@ -0,0 +1,12 @@
+include "application.conf"
```

Comment:
*Question(non-blocking)*: This is going to start the database container for every test, including if you don't need it.  Is that worth it, do you think?  I'm not sure personally.  If you think so, LGTM!

Reply:
oo that's true. I didn't think about that.

I'm gonna script and run some timing tests to see how much time this actually costs.